### ISSUE TITLE
Validation dataset used

### ISSUE THREAD

**Issue by LilJordan23** [seongq/flowmse#2]
Thanks for your contribution to this field, which inspiring me a lot. In the training process, you spilt part of trainset of VB-Demand as validation set, Could you please answering the following queations.
1 how to explain the potiential change of data preferance when training with a not complete data, which is spilted into train and validation 2 parts.
2  Is all the experiences you compared with using the same valid logic?
3  Why not just pretrain by using the loss function?
I would appreciate it if you can reply, thanks!

**Comment by seongq** [comment]
Thanks for your comments.

1. Good question. But I don't know, there are many situations, so I can not give you clear answers.
2. I used the same valid logic for FlowSE.
3. I simply applied flow-matching and interpreted FlowSE as diffusion model in my paper. Pre-train and fine-tuning is worth doing for further researches.
